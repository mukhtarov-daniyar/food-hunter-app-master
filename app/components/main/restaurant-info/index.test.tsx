import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import RestaurantInfo from './index';

it('renders correctly', () => {
  const tree = renderer.create(<RestaurantInfo />).toJSON();
  expect(tree).toMatchSnapshot();
});
