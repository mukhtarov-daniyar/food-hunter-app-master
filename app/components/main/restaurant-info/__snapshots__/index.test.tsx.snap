// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "paddingHorizontal": 19.23076923076923,
    }
  }
>
  <Text
    style={
      {
        "color": "#3D3D3D",
        "fontSize": 41.666666666666664,
        "fontWeight": "bold",
        "marginBottom": 38.46153846153846,
      }
    }
  />
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        [
          {
            "color": "#FF5252",
            "fontSize": 25,
          },
          undefined,
          {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          {},
        ]
      }
    >
      ïŽƒ
    </Text>
    <Text
      style={
        {
          "color": "#3D3D3D",
          "fontSize": 33.333333333333336,
          "marginLeft": 9.615384615384615,
        }
      }
    />
  </View>
</View>
`;
